src/system/system_win.ml

```diff
@@ -38,10 +38,13 @@ let fspathAddSuffixToFinalName f suffix = f ^ suffix
 (****)
 
 let fixPath f =
-  for i = 0 to String.length f - 1 do
-    if f.[i] = '/' then f.[i] <- '\\'
-  done;
-  f
+  let length = String.length f in
+  let buffer = Bytes.create length in
+    String.blit f 0 buffer 0 length;
+    for i = 0 to length - 1 do
+      if Bytes.get buffer i = '/' then Bytes.set buffer i '\\'
+    done;
```

Comment:
The indentation is confusing here, all these lines should be indented the same way.

Reply:
I pushed a "fix!" commit changing this... if it's not the correct indentation, please post a "suggested change" and I'll update it.

Reply:
It is good now.